spec/services/proofing/socure/id_plus/request_spec.rb

```diff
@@ -0,0 +1,243 @@
+require 'rails_helper'
+
+RSpec.describe Proofing::Socure::IdPlus::Request do
+  let(:config) do
+    Proofing::Socure::IdPlus::Config.new(
+      api_key:,
+      base_url:,
+      timeout:,
+    )
+  end
+
+  let(:api_key) { 'super-$ecret' }
+
+  let(:base_url) { 'https://example.org/' }
+
+  let(:timeout) { 5 }
+
+  let(:user) { build(:user) }
+
+  let(:input) do
```

Comment:
personal style, I'd remove the empty liens
```suggestion
  let(:api_key) { 'super-$ecret' }
  let(:base_url) { 'https://example.org/' }
  let(:timeout) { 5 }
  let(:user) { build(:user) }
  let(:input) do
```